src/TraceEvent/TraceEvent.csproj

```diff
@@ -41,7 +40,7 @@
     <WarningLevel>3</WarningLevel>
     <DocumentationFile>bin\Debug\Microsoft.Diagnostics.Tracing.TraceEvent.xml</DocumentationFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>0649 0618</NoWarn>
+    <NoWarn>0649,0618</NoWarn>
```

Comment:
As mentioned if we can't get rid of them other ways, I would prefer the blanked NoWarn on the obsolete flag over many #pragma no warns.